source/process/definitions.rst

```diff
@@ -0,0 +1,58 @@
+.. _definitions:
+
+Definitions
+===========
+
+
+Feature Quality Levels
+----------------------
+
+Production Level Quality
+  - Recommended for use in production environments
+  - Supported in Mattermost, Inc. SLA
+  - Documented
+  - Included in security reviews
+  - Tested on several platforms
+Beta Level Quality
+  - Is a work in progress
+  - Not recommended in production without `Premier Support <https://about.mattermost.com/support/>`_ engagement with Mattermost, Inc.
+  - Minimal documentation
+  - Disabled by default and must be turned on with a run-time feature flag
+  - Tested thoroughly on a small set of platforms, with tickets and work items drafted, and made available for testing and feedback on additional platforms
+Experimental Level Quality
+  - Not recommended for use in production
+  - Unknown level of stability, and might cause data loss
```

Comment:
It means data loss due to untested, possibly buggy experimental code that inadvertently deletes or otherwise mangles messages and other information, either in the database or on disk.